### DISCUSSION TITLE
Welcome to patchfox Discussions!

### DISCUSSION THREAD

**Discussion by soapdog** [soapdog/patchfox#146] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bhaugen** [comment]
Hi soapdog! It's a-me, Bob Haugen from Patchwork, which is not working anymore so I need to explore other SSB clients.

Tried PatchBay, which I had insralled awhile ago. Have not been able to read much of anything there yet. Font too small and light, eyes too old. Tried to change font, failed. 

But PatchFox seems like it might be simpler anyway. Runs in Firefox! Right?

But I I don't know what to do first and second and third. Got a Getting Started doc?

**Comment by soapdog** [comment]
Hi @bhaugen,

The version of Patchfox that runs on Firefox or Chrome should work well for you, but it is not a standalone version, it needs a running client such as Patchwork to piggy back. If Patchwork is not working for you, then I don't know if I can make Patchfox work. 

Have you tried https://manyver.se/ yet? It is probably the most up to date client and should work well for you. It works on desktops now.

**Reply by bhaugen** [reply]
Thanks @soapdog - I think the problem with Patchwork is at least mostly in the UI, so the backend might run and feed Patchfox. Or at least I am up for trying it.

But that's good news that Manyverse now runs on desktops. I suspect the problem there will be compatibility with the SSB network. Don't know if you can read this: 
https://social.coop/@staltz@mastodon.social/110151665185148755 
but Andre Staltz says,
" It won't be interoperable with SSB, it's new beginnings."

**Reply by soapdog** [reply]
@bhaugen that will be for the new thing Staltz will build. The current ManyVerse is SSB and compatible with all the network.
